### PR TITLE
Release `anaconda-client` version 1.13.0

### PR DESCRIPTION
**Pull request by mattkram**
This PR contains all changes for `anaconda-client` v1.13.0 release. Please see the `CHANGELOG.md` for more details.

### CODE REVIEW

File: conda.recipe/meta.yaml [hunk 1]
```diff
@@ -5,25 +5,26 @@ package:
...
   script: {{ PYTHON }} -m pip install --no-build-isolation --no-deps .
+  noarch: python
   entry_points:
-    - anaconda = binstar_client.scripts.cli:main
     - binstar = binstar_client.scripts.cli:main
     - conda-server = binstar_client.scripts.cli:main
 
 requirements:
-  host:
+  host:
```
Comment (mattkram) [inline conda.recipe/meta.yaml:19]:
```suggestion
  host:
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,34 @@
 
 We [keep a changelog.](http://keepachangelog.com/)
 
+## 1.13.0
+
+As part of extending our CLI, we have been migrating to a plugin-based system. 
+This release removes the `anaconda` entrypoint from `anaconda-client`, which is now located instead in `anaconda-cli-base`.
+In order to maintain backwards compatibility for all subcommands, we include logic to mount all existing subcommands from `anaconda-client` as top-level subcommands.
```
Comment (AlbertDeFusco) [inline CHANGELOG.md:9]: how about "... as hidden top-level subcommands. anaconda-client subcommands continue to work until overridden by new plugins, but will not be displayed in the help output"

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,34 @@
...
+
+In the case that another plugin is installed (like `anaconda-cloud-auth`), the following changes may be observed:
+
+* All existing `anaconda-client` subcommands are available
+* All existing `anaconda-client` subcommands are ALSO available with the `org` prefix, e.g. `anaconda org upload`. 
+  It is recommended to adopt this format, as it is more explicit.
+* The help text will be modified (i.e. type `anaconda`). 
+  Not all `anaconda-client` subcommands will be shown.
+* The `anaconda notebooks` subcommand will explicitly be listed as deprecated in the CLI help.
+
```
Comment (AlbertDeFusco) [inline CHANGELOG.md:23]: Maybe include a quick statement about forcing legacy behavior?

File: binstar_client/commands/download.py [hunk 4]
```diff
@@ -32,7 +32,7 @@ def add_parser(subparsers):
 
     parser.add_argument(
         'handle',
-        help='user/notebook',
+        help='user/package_name',
```
Comment (AlbertDeFusco) [inline binstar_client/commands/download.py:35]: maybe write this as `<channel_name>/<package_name>` like the docstring above?

File: conda.recipe/meta.yaml [hunk 5]
```diff
@@ -33,9 +34,9 @@ requirements:
     - requests >=2.20.0
     - requests-toolbelt >=0.9.1
     - setuptools >=58.0.4
-    - six >=1.15.0
     - tqdm >=4.56.0
     - urllib3 >=1.26.4
+    - anaconda-cli-base >=0.3.0
```
Comment (AlbertDeFusco) [inline conda.recipe/meta.yaml:39]: maybe bump this to `>=0.4`?

File: requirements.txt [hunk 6]
```diff
@@ -11,6 +11,6 @@ platformdirs>=3.10.0,<5.0
 requests>=2.20.0
 requests-toolbelt>=0.9.1
 setuptools>=58.0.4
-six>=1.15.0
 tqdm>=4.56.0
 urllib3>=1.26.4
+anaconda-cli-base>=0.3.0
```
Comment (AlbertDeFusco) [inline requirements.txt:16]: again bump to `>=0.4`?

**Review by AlbertDeFusco — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (mattkram) [reply CHANGELOG.md:20]: Added a sentence below

File: CHANGELOG.md (hunk 2 above)
Comment (mattkram) [reply CHANGELOG.md]: Thank you, I've modified the text accordingly.

File: binstar_client/commands/download.py (hunk 4 above)
Comment (mattkram) [reply binstar_client/commands/download.py]: I've updated this.

File: requirements.txt (hunk 6 above)
Comment (mattkram) [reply requirements.txt]: Good call, done.

File: conda.recipe/meta.yaml (hunk 5 above)
Comment (mattkram) [reply conda.recipe/meta.yaml]: Done.

**Pull request merged**